tool/dart_dev/config.dart

```diff
@@ -3,5 +3,8 @@ import 'package:glob/glob.dart';
 
 final config = {
   ...coreConfig,
-  'format': FormatTool()..exclude = [Glob('test/**/fixtures/**.dart')],
+  'analyze': AnalyzeTool()..useDartAnalyze = true,
+  'format': FormatTool()
+    ..exclude = [Glob('test/**/fixtures/**.dart')]
+    ..formatter = Formatter.dartFormat,
```

Comment:
Do we want these for ourselves? I assumed so but I'm open to changing them back.